storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -610,18 +610,26 @@ uint32_t Archived_Redo_Log_Monitor::get_first_log_block_checksum() const {
   return first_log_block_checksum;
 }
 
-void Archived_Redo_Log_Monitor::skip_for_block(lsn_t lsn, uint32_t no,
-                                               uint32_t checksum) {
+void Archived_Redo_Log_Monitor::skip_for_block(lsn_t lsn, const byte *buf) {
```

Comment:
instead of buf, can we use more meaningful name? is it redo block, archive block buffer?